README.md

```diff
@@ -89,6 +103,76 @@ What do you need to do:
 
 For example: "Maven Build" steps have path `modules/Build/MavenBuild.groovy` and placed in the library - feel free to check it out.
 
+### Post Steps
+
+MPL supports 2 useful poststep interfaces which allow you to store all the logic of module in the same file.
+
+All the poststeps will be executed in LIFO order and all the exceptions will be collected & displayed in the logs.
+
+#### MPLModulePostStep
+
+Allow to set some actions that need to be executed right after the current module (doesn't matter it fails or not).
+
+Could be useful when you need to collect reports or clean stage agent before it will be killed.
+
+If module post step fails - it's fatal for the module, so the pipeline will fail (unlike general poststeps). All the poststeps
+for the module will be executed and errors will be printed, but module will fail.
+
+`{NestedLibModules}/Build/MavenBuild.groovy`:
```

Comment:
Looks like it should be:
`{NestedLibModules}/Build/Build.groovy`:

Reply:
Not necessarily - we can override any module and after that call the same name and it will be executed from the upper-level library. So call path will be like:
* `{MPLModules}/Build/Build.groovy`
  * `{NestedLibModules}/Build/MavenBuild.groovy`
    * `{MPLModules}/Build/MavenBuild.groovy`
  * `ModulePostStep:` `{NestedLibModules}/Build/MavenBuild.groovy`